torch/_inductor/codegen/triton.py

```diff
@@ -1662,7 +1679,7 @@ def call_kernel(self, code, name: str):
             )
         else:
             # TODO: refactor generate_kernel_call
-            call_args_str = ", ".join(call_args)
+            call_args_str = ", ".join(str(item) for item in call_args)
```

Comment:
`str` here because args can be symbolic expressions? Can you leave a comment?

Reply:
Yes, will add a comment.